etc/config/d.amazon.properties

```diff
@@ -1,5 +1,5 @@
 compilers=&gdc:&ldc:&dmd
-defaultCompiler=ldc1_26
+defaultCompiler=ldc1_27
 objdumper=/opt/compiler-explorer/gcc-11.1.0/bin/objdump
 demangler=/opt/compiler-explorer/ldc1.21.0/ldc2-1.21.0-linux-x86_64/bin/ddemangle
```

Comment:
Should the demangler be updated?

Reply:
Don't think there were any changes, but it won't hurt. Updated it to 1.27 too.